res/css/views/elements/_RichText.scss

```diff
@@ -6,16 +6,23 @@
 .mx_RoomPill,
 .mx_GroupPill,
 .mx_AtRoomPill {
-    border-radius: 16px;
-    display: inline-block;
-    height: 20px;
+    display: inline-flex;
+    align-items: center;
+    vertical-align: middle;
+    border-radius: $font-16px;
+    height: $font-16px;
     line-height: $font-20px;
-    padding-left: 5px;
+    padding-left: 0;
+    padding-right: 0.5em;
```

Comment:
The usage of `em` makes it a bit hard to follow with what this scales with, as there is no font-size set on this it seems, so probably with some parent? Could we convert this to `rem` so it's more obvious what size this corresponds to?

Reply:
Done